lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -110,8 +110,8 @@ function dosomething_user_menu_alter(&$items) {
   $items['user/%'] = array(
     'page callback' => 'dosomething_user_get_view',
     'page arguments' => array(1),
-    'access callback' => 'user_access',
-    'access arguments' => array('access user profiles'),
+    'access callback' => 'user_view_access',
```

Comment:
hmm, does this expose more than we want though? can users see `edit` now instead of `edit info` for instance?